lms/templates/courseware/gradebook.html

```diff
@@ -74,13 +74,21 @@
         <thead>
           <tr> <!-- Header Row -->
             %for section in templateSummary['section_breakdown']:
-              <th><div class="assignment-label">${section['label']}</div></th>
+              <%
+                tooltip_str = section['detail']
+                if "=" in section['detail']:
```

Comment:
Hi,

Beacuse we made header label from the first student record,  i was adding
tool tip (title)  to header column.  There is a field ```detail``` in
student record. So i am processing this string here to remove student
specific data i.e marks, percentage etc and get the homework detail (title
of home work) from this student record.
On 03-Oct-2015 2:39 am, "Sarina Canelake" <notifications@github.com> wrote:

> In lms/templates/courseware/gradebook.html
> <https://github.com/edx/edx-platform/pull/10021#discussion_r41070606>:
>
> > @@ -74,13 +74,21 @@
> >          <thead>
> >            <tr> <!-- Header Row -->
> >              %for section in templateSummary['section_breakdown']:
> > -              <th><div class="assignment-label">${section['label']}</div></th>
> > +              <%
> > +                tooltip_str = section['detail']
> > +                if "=" in section['detail']:
>
> ?? What is this condition? I'm very confused here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10021/files#r41070606>.
>